<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="startLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>开机启动</string>
   </property>
  </widget>
  <widget class="QLabel" name="softwareLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>52</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;楷体&quot;;
color: rgb(255, 255, 255);

</string>
   </property>
   <property name="text">
    <string>启用软件</string>
   </property>
  </widget>
  <widget class="QLabel" name="waketimeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>提醒时间间隔</string>
   </property>
  </widget>
  <widget class="QLabel" name="restLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>246</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>小憩休息时长</string>
   </property>
  </widget>
  <widget class="QToolButton" name="minBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>30</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{  
	border-image: url(:/images/Widget/min.png);
} 

QToolButton:hover{  
	border-image: url(:/images/Widget/min_hover.png);
} 
QToolButton:pressed{  
	border-image: url(:/images/Widget/min_pressed.png);
}  
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>30</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{  
	border-image: url(:/images/Widget/close.png);
} 

QToolButton:hover{  
	border-image: url(:/images/Widget/close_hover.png);
} 
QToolButton:pressed{  
	border-image: url(:/images/Widget/close_pressed.png);
}  
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="BKLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#212121;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="timeLine">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>203</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;楷体&quot;;

    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 0 #646464, stop: 1 #5d5d5d);
    padding: 1px;
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;</string>
   </property>
   <property name="text">
    <string>30</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="restLine">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;楷体&quot;;
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 0 #646464, stop: 1 #5d5d5d);
    padding: 1px;
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QToolButton" name="use_softwareBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>39</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/Widget/on_key.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="start_upBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/Widget/on_key.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="minLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #607d8b;
font: 14pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>分</string>
   </property>
  </widget>
  <widget class="QLabel" name="min2Label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>245</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #607d8b;
font: 14pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>分</string>
   </property>
  </widget>
  <widget class="QLabel" name="logoLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>276</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>by:无木成林</string>
   </property>
  </widget>
  <widget class="QLabel" name="addLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>扣电池版</string>
   </property>
  </widget>
  <widget class="QLabel" name="lockskinLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>选择本地皮肤</string>
   </property>
  </widget>
  <widget class="QToolButton" name="lock_versionBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/Widget/on_key.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lockskinBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>165</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    color: #b1b1b1;
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #565656, stop: 0.1 #525252, stop: 0.5 #4e4e4e, stop: 0.9 #4a4a4a, stop: 1 #464646);
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 6;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>420</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #000000;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>79</y>
     <width>410</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#484848;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>120</y>
     <width>410</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#484848;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>160</y>
     <width>410</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#484848;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>196</y>
     <width>410</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#484848;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>234</y>
     <width>410</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#484848;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>272</y>
     <width>410</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#484848;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>420</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#484848;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>BKLabel</zorder>
  <zorder>label_2</zorder>
  <zorder>startLabel</zorder>
  <zorder>softwareLabel</zorder>
  <zorder>waketimeLabel</zorder>
  <zorder>restLabel</zorder>
  <zorder>minBtn</zorder>
  <zorder>closeBtn</zorder>
  <zorder>timeLine</zorder>
  <zorder>restLine</zorder>
  <zorder>use_softwareBtn</zorder>
  <zorder>start_upBtn</zorder>
  <zorder>minLabel</zorder>
  <zorder>min2Label</zorder>
  <zorder>logoLabel</zorder>
  <zorder>addLabel</zorder>
  <zorder>lockskinLabel</zorder>
  <zorder>lock_versionBtn</zorder>
  <zorder>lockskinBtn</zorder>
  <zorder>line_2</zorder>
  <zorder>line_3</zorder>
  <zorder>line_4</zorder>
  <zorder>line_5</zorder>
  <zorder>line_6</zorder>
  <zorder>line_7</zorder>
  <zorder>line_8</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
